### ISSUE TITLE
Show user smallest potential file size

### ISSUE THREAD

**Issue by mrmrs** [cssstats/cssstats#148]
Show a user a potential file size if every unique css rule in the original css was made into single purpose classes. 

i.e All duplicate rules will be removed from the original css source.

Show full and gzipped file sizes as well as delta between potential values and original values.

This will help illustrate what the bare minimum file size is to replicate all of their current visual styles.

**Comment by jxnblk** [comment]
I assume you mean combining the duplicative declarations into unique rulesets, right?

Do we want to have something that also shows potential for normalizing the values as well? (E.g. 13px -> 14px)

**Comment by mrmrs** [comment]
@jxnblk you're correct. Updated the issue description.

**Comment by necolas** [comment]
Yeah I wanted to know how many unique declarations exist in a CSS file. Could you add that to the reported stats? I'd like to see if most larger websites have comparable numbers of unique declarations.

**Comment by mrmrs** [comment]
@necolas - that is EXACTLY what I want to try to visualize / show. Working on it. I think we have a different issue for that but let me double check.

**Comment by necolas** [comment]
I hacked it in on a local version of the site. Some sites, like twitter, have 2500+ unique declarations. Most of that is a huge number of different lengths used for all parts of the box model and position. Other sites are closer to 500–1000 unique declarations. But most of the time the number of unique declarations is around or much lower than the number of selectors or declarations.

**Comment by mrmrs** [comment]
Was working on adding a view for just unique declarations and figured I'd take a stab at calculating smaller possible file size. 
I figured I'd give a few numbers to see the range. This shows: 
1. All unique declarations as atomic classes
2. All unique declarations as atomic classes name spaced across 3 breakpoints (4 variants total) 
3. All unique declarations as atomic classes name spaced across all media queries in the CSS

<img width="936" height="416" alt="Image" src="https://github.com/user-attachments/assets/40c802df-b547-4bdf-af37-c4510ccb6106" />
